File: app/src/main/java/com/zwstudio/logicpuzzlesandroid/puzzles/carpenterswall/domain/CarpentersWallCornerObject.java
package com.zwstudio.logicpuzzlesandroid.puzzles.carpenterswall.domain;

import com.zwstudio.logicpuzzlesandroid.home.domain.HintState;

public class CarpentersWallCornerObject extends CarpentersWallObject {
    public HintState state = HintState.Normal;
    public int tiles;
    public boolean isHint() {return true;}
    public String objAsString() {
        return "corner";
    }
}

File: app/src/main/java/com/zwstudio/logicpuzzlesandroid/puzzles/carpenterswall/domain/CarpentersWallCornerObject.kt
package com.zwstudio.logicpuzzlesandroid.puzzles.carpenterswall.domain

import com.zwstudio.logicpuzzlesandroid.home.domain.HintState

open class CarpentersWallCornerObject : CarpentersWallObject() {
    var state = HintState.Normal
    var tiles = 0
    override val isHint: Boolean
        get() = true

    override fun objAsString(): String {
        return "corner"
    }
}